website/public/locales/de/common.json

```diff
@@ -1,36 +1,61 @@
 {
   "about": "Über",
   "account_settings": "Konto",
-  "admin_dashboard": "Admin-Bereich",
+  "admin_dashboard": "Admin-Dashboard",
+  "back_to_dashboard": "Zum Dashboard zurückkehren",
+  "bug_description": "Wenn Sie Daten beitragen wollten aber hier gelandet bist, melde bitte einen Fehler.",
+  "bug_button": "Fehler Melden",
+  "cancel": "Abbrechen",
+  "chat": "Chat",
+  "confirm": "Bestätigen",
   "connect": "Kontakt aufnehmen",
   "conversational": "Konversations-KI für alle.",
   "copied": "Kopiert",
+  "create_chat": "Neuen Chat starten",
   "dark_mode": "Dunkler Modus",
-  "dashboard_home": "Dashboard Home",
   "dashboard": "Dashboard",
+  "dashboard_home": "Dashboard Home",
   "delete": "Löschen",
   "discord": "Discord",
   "docs": "Doku",
+  "edit": "Bearbeiten",
+  "faq": "FAQ",
+  "guidelines": "Richtlinien",
   "github": "GitHub",
   "leaderboard": "Leaderboard",
   "legal": "Rechtliches",
   "light_mode": "Heller Modus",
   "loading": "Wird geladen...",
-  "messages_dashboard": "Messages Dashboard",
   "messages": "Nachrichten",
+  "messages_dashboard": "Nachrichten-Dashboard",
   "more_information": "Weitere Informationen",
   "no": "Nein",
+  "output": "Ausgabe",
+  "parameters": "Parameter",
   "privacy_policy": "Datenschutz-Bestimmungen",
+  "prompt": "Prompt",
   "report_a_bug": "Einen Fehler melden",
+  "retry": "Erneut Versuchen",
+  "review": "Review",
+  "send": "Senden",
+  "sorry_404": "Sorry, die Seite nach der sie suchen existiert nicht.",
```

Comment:
"Sie" .. in general all messages should be either address the user with formal "Sie" or "Du" .. currently it seems to be mixed, e.g. see "Deine Antwort abschicken"

Reply:
Ah, yea. I used the informal at first before noticing that the existing translations were using formal language so I missed this when switching it over.